src/test/java/seedu/linkedout/logic/commands/SearchCommandTest.java

```diff
@@ -90,6 +93,32 @@ public void execute_multipleKeywords_multipleApplicantsFound() {
         assertEquals(Arrays.asList(CARL, ELLE, FIONA), model.getDefaultApplicantList());
     }
 
+    @Test
+    public void execute_multipleSkills_applicantFound() {
+        String expectedMessage = String.format(MESSAGE_APPLICANTS_LISTED_OVERVIEW, 1);
+        ApplicantContainsSkillKeywordsPredicate predicate = prepareSkillPredicate("Photography Videography");
+        List<KeywordsPredicate> keywordPredicate = new ArrayList<>();
+        keywordPredicate.add(predicate);
+        SearchCommand command = new SearchCommand(keywordPredicate);
+        expectedModel.updateSearchedApplicantList(keywordPredicate);
+        assertCommandSuccess(command, model, expectedMessage, expectedModel);
+        assertEquals(Arrays.asList(BENSON), model.getDefaultApplicantList());
+    }
+
+    @Test
+    public void execute_multipleAttributes_applicantFound() {
+        String expectedMessage = String.format(MESSAGE_APPLICANTS_LISTED_OVERVIEW, 1);
+        ApplicantContainsSkillKeywordsPredicate predicate = prepareSkillPredicate("Photography Videography");
+        NameContainsKeywordsPredicate predicateTwo = prepareNamePredicate("Meier");
```

Comment:
Since there are multiple predicate, can consider following the practice above, `skillPredicate` `namePredicate` for eg.

Reply:
Changed!